chart/values.yaml

```diff
@@ -578,8 +576,8 @@ loki-stack:
       runAsUser: 1001
     # initContainers to chown the static hostpath PV by 1001 user
     initContainers:
-      - command: ["/bin/bash", "-ec", "chown -R 1001:1001 /data"]
-        image: docker.io/bitnami/bitnami-shell:10
+      - command: ["/bin/sh", "-ec", "chown -R 1001:1001 /data"]
+        image: docker.io/busybox:1.36.1-musl
```

Comment:
That's very appropriate :O

Reply:
though I wonder if we should put our own on openebs's dockerhub?
Sadly for etcd, their init container has the init hardcoded to use bash IIRC, so we'd still need one with bash.

Reply:
Sure, I can see the utility in that. For someone behind an air-gap, they might be taking stock of all the containers they'd be needing.
This one already lives on the openebs org: https://github.com/openebs/linux-utils/blob/main/Dockerfile
We could add bash to the apk list. Also it's alpine, so it'd have musl. But even if it didn't, everything we build is statically built and our shell scripts don't really depend on the libc implemention differences of musl/glibc, etc.